tsnet/tsnet.go

```diff
@@ -323,6 +323,17 @@ func (s *Server) doInit() {
 	}
 }
 
+func (s *Server) TailscaleIPs() []netip.Addr {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	defer cancel()
+
+	st, err := s.localClient.Status(ctx)
```

Comment:
This is much heavier than you need. StatusWithoutPeers is cheaper but still not as cheap as calling NetMap method on s.lb

Reply:
Done.